src/interface/swagger.json

```diff
@@ -984,7 +984,7 @@
     "Amount": {
       "title": "Amount",
       "type": "string",
-      "pattern": "^([0]|([1-9][0-9]{0,17}))([.][0-9]{0,3}[1-9])?$",
+      "pattern": "^(([1-9][0-9]{0,17}))([.][0-9]{0,3}[1-9])?$",
```

Comment:
@gibaros @sri-miriyala Except this change, everything else is fine...

The specification 'Amount' type allows 'zero' so lets defer this for now (do not include it for the time being).. It may be a scheme level error but according to the API Specification, it is not a bug to allow a zero amount..

I think the onus/responsibility is on the FSPs / Scheme to make sure that these kind of reqeusts are not made, however, if we want the Switch to validate, then it should be part of internal validation and not Schema validation as  such..